peekingduck/pipeline/nodes/model/jdev1/jde_files/tracker.py

```diff
@@ -142,11 +142,11 @@ def track_objects_from_image(
                 online_ids.append(target.track_id)
                 scores.append(target.score.item())
         if not online_tlwhs:
-            return online_tlwhs, online_ids, scores
+            return np.empty((0, 4)), online_ids, np.empty(0)
```

Comment:
c.f. `movenetv1/movenet_files/predictor.py` comment below.

Reply:
previously this returned empty `List` instead of empty numpy arrays. In additional to the benefit of a guaranteed shape, ensuring this has a numpy array type prevents code using `arr.shape` or `arr.size` to check for emptiness from breaking.